oat-sa/extension-tao-test-runner-plugins
# fix/TR-187 fix full screen handler in IE11

Related to https://oat-sa.atlassian.net/browse/TR-187

accepted solution is
"Think the default dialog can be customized. For IE11, we could replace the confirm dialog with a variant saying something like: The assessment must be taken in fullscreen. Please press F11 to activate it., and then replace the ok button with a cancel. 
No more programmatical call to setFullscreen() in this context but only a check to see if the mode is activated. The dialog will show up until the fullscreen mode is detected."

To see cancel button label https://github.com/oat-sa/tao-core-ui-fe/pull/264 is needed

to test:
1) checkout
2) change config debug mode to false in /config/generis.conf.php:50 to define('DEBUG_MODE', false);
3) Run test in IE11 with enabled security (NFER project) but make sure your dev toolbar is closed (beware of cache) with dev toolbar opened full-screen wouldn't be detected.
4) click ok or hit enter should close and open alert (no full screen triggered) but clicking F11 fill trigger fullscreen and dialog disappear.
5) click F11 again (you may need to click it twice(double click), existing bug, still looking roots for it)
6) alert dialog should appear again

ps. make sure other browsers keep a previous behavior (click/hit enter will trigger a fullscreen)

- [ ] need to be merged after https://github.com/oat-sa/tao-core-ui-fe/pull/264
